scripts/sfpshow

```diff
@@ -391,15 +393,18 @@ class SFPShow(object):
         output = ''
 
         sfp_info_dict = state_db.get_all(state_db.STATE_DB, 'TRANSCEIVER_INFO|{}'.format(interface_name))
-        output = interface_name + ': ' + 'SFP EEPROM detected' + '\n'
-        sfp_info_output = self.convert_sfp_info_to_output_string(sfp_info_dict)
-        output += sfp_info_output
+        if sfp_info_dict['type'] == RJ45_PORT_TYPE:
```

Comment:
where is port type updated for sfp_info_dict[] ? Also, how this will be handled in platforms using SFP-refactor

Reply:
@prgeor 
when xcvrd starts it fetches all transceivers’ information and push the data into TRANSCEIVER_INFO table. This is the place where it is initialized. After this point it will be never changed.
When an RJ45 port is unplugged, an error event will be reported instead of an unplugged event, which guarantees the type in TRANSCEIVER_INFO table will not be changed. Xcvrd push the error status into TRANSCEIVER_STATE table as the present info. Sfp utility will fetch the info to display the present status.
SFP refactor has been used for Mellanox platforms. It doesn’t affect the way it is handled.
Hopefully your questions are answered